app/views/campaigns/programs.html.haml

```diff
@@ -67,5 +67,9 @@
             %th{style: 'width:100px'}
               = t('courses.actions')
       %tbody.list
+      - if @keywords.nil?
```

Comment:
Rather than if-check a negative, the standard Ruby way to handle this would be to switch the order and do
```
if @keywords.present?
  <do search version>
else
  <do normal version>
```